parkhangs-webapp/src/features/modal/ModalParkDetail.js

```diff
@@ -3,14 +3,26 @@ import {deleteEvent} from 'features/events/eventsSlice'
 import {connect} from 'react-redux'
 import 'features/modal/ModalParkDetail.css'
 import moment from 'moment'
+import CommentForm from 'components/CommentForm'
+import { fetchParkComments, deleteParkComment } from 'features/comments/commentSlice'
 import NoFilledHeartIcon from 'assets/icons/heart-no-fill.svg'
 import FilledHeartIcon from 'assets/icons/heart-filled.svg'
 import { toggleSavedPark } from "features/users/userSlice";
 import AddToCalendar from 'react-add-to-calendar';
+import { addRating } from 'features/parks/parksSlice'
 import ShareCalendar from 'components/ShareCalendar'
 
 
 class ModalParkDetail extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.toggleFavouritePark = this.toggleFavouritePark.bind(this);
```

Comment:
not sure this is necessary since both of these methods use arrow functions

Reply:
this may be from an old commit? I didn't write this constructor, I got it when I pulled from project 5 awhile ago.